### ISSUE TITLE
Suggestion: jsfiddle

### ISSUE THREAD

**Issue by shaneray** [ManrajGrover/SingleDivProject#28]
Might be nice to include a jsfiddle or something similar in the readme.

Example: https://jsfiddle.net/xk48313a/

**Comment by module17** [comment]
Agreed; it is nicer. Why the Assets/*.png too?

**Comment by ManrajGrover** [comment]
@shaneray Thank you for suggesting this. Yes, we definitely need a page to showcase this. Allowing jsfiddle and codepen linking would be awesome. Do you want it to be added as 4th column or replacement of Assets? :blush: 

@module17 `Assets/*png` was something that was being updated before the popularity of the project. Due to sudden attention, this was tough to change. Also, gh-page or something similar will take a while to come up. Hence, to showcase everything at one place, assets is required for now. :blush:

**Comment by shaneray** [comment]
It is nice to see the image at a glance, so I would add it as a 4th column.  This also seems like it falls in line with some of the other suggestions of making the image sizes consistent and using github pages.

**Comment by module17** [comment]
@shaneray Yes, exactly. Image sizes and formats need more consistency. The correct flag proportions should be enforced as well.
